deepchecks/vision/checks/distribution/train_test_label_drift.py

```diff
@@ -65,7 +70,7 @@ class TrainTestLabelDrift(TrainTestCheck):
         out of the EMD calculation. This is done in order for extreme values not to affect the calculation
         disproportionally. This filter is applied to both distributions, in both margins.
     max_num_categories_for_drift: int, default: 10
-        Only for non-continuous properties. Max number of allowed categories. If there are more,
+        Only for categorical properties. Max number of allowed categories. If there are more,
```

Comment:
For now, write "discrete". In the next task where we rewrite the "output_type" thingy then we'll change this docstring as well